yubikey-u2f-pam-plugin/auth-pam-u2f.c

```diff
@@ -902,14 +903,13 @@ pam_server(int fd, const char *service, int verb, const struct name_value_list *
 /// 1 = Error
 /// 2 = Return client reason
 static int
-u2f_auth_verify(char *username, char* password, char* script_path, char **client_reason)
+u2f_auth_verify(char *username, char* password, char* script_path, char *client_reason)
```

Comment:
Have the caller pass a pointer to a char array where we are to store the client reason.